### DISCUSSION TITLE
cefsharp webbrowser seed

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#3548] [Ideas]
I see that there are basic, very basic examples ... it is possible to contribute something a little more intermediate or advanced. something like an example of a web browser? I am interested in helping.

**Comment by amaitland** [comment]
The https://github.com/cefsharp/CefSharp/tree/master/CefSharp.WinForms.Example and https://github.com/cefsharp/CefSharp/tree/master/CefSharp.Wpf.Example projects  both provide basic tabbed browser functionality, your welcome to contribute enhancements to one or both of those.

There are already a number of open source browsers here on GitHub using CefSharp. 

- https://github.com/sharpbrowser/SharpBrowser
- https://github.com/Haltroy/Korot
- https://github.com/erfg12/AOL_4.0_Emu

Probably a couple more I just don't remember their names.

**Reply by rover886** [reply]
@arcanisgk By using CefSharp I have developed a Secured Browser for an examination system. That browser have whole lot of functionalities on top of browsing. Like,

- Capturing images applying face detection and recognition
- Capturing sounds to check if anyone is in the room 
- Restricts switching between the applications
- Restricting remote access to the machine
- Also, when the Secured Browser is opened it will block all internet connection other than exam website.
